### PR TITLE
Edit typos in 'Try Swarm at scale'

### PR DESCRIPTION
**Pull request by ogryzek**
Signed-off-by: Drew Ogryzek <drew@drewbro.com>

### CODE REVIEW

**Comment by MHBauer** [comment]
This all looks good to me.

File: docs/swarm_at_scale/about.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Behind the frontend is a worker tier which runs on separate nodes. This tier:
 Just like the frontend, the worker tier can also scale arbitrarily. The worker
 count and frontend count are independent from each other.
 
-The applications dockerized microservices are deployed to a container network.
+The application's dockerized microservices are deployed to a container network.
```
Comment (nishanttotla) [inline docs/swarm_at_scale/about.md]: @ogryzek while you're at it, could you also change `dockerized` -> `Dockerized`?

File: docs/swarm_at_scale/deploy-app.md [hunk 2]
```diff
@@ -139,7 +139,7 @@ command below, look for the value constraint.
     --name results-app docker/example-voting-app-result-app
     ```
 
-7. Start voting application twice, on each frontend node.
+7. Start the voting application twice. Once on each frontend node.
```
Comment (nishanttotla) [inline docs/swarm_at_scale/deploy-app.md]: I think the comma makes more sense here, including the addition of `once`

**Comment by nishanttotla** [comment]
Two suggestions, otherwise LGTM

**Comment by dongluochen** [comment]
Change LGTM. 

@SvenDowideit Please take a look at the documentation test. Thanks!

**Comment by SvenDowideit** [comment]
docs LGTM - @thaJeztah 

@dongluochen @ogryzek yup - the ERROR's should be resolved by https://github.com/docker/swarm/pull/2185

you guys can either merge as is, or rebase to pick up the new Dockerfile :)

**Comment by dongluochen** [comment]
Thank you @SvenDowideit .

**Pull request merged**